netkan/netkan/cli.py

```diff
@@ -309,12 +310,77 @@ def ticket_closer(token, days_limit):
 )
 def auto_freezer(netkan_remote, token, repo, user, days_limit, key):
     init_ssh(key, '/home/netkan/.ssh')
-    af = AutoFreezer(
+    afr = AutoFreezer(
         init_repo(netkan_remote, '/tmp/NetKAN'),
         GitHubPR(token, repo, user)
     )
-    af.freeze_idle_mods(days_limit)
-    af.mark_frozen_mods()
+    afr.freeze_idle_mods(days_limit)
+    afr.mark_frozen_mods()
+
+
+@click.command()
+@click.option(
+    '--queue', envvar='SQS_QUEUE',
+    help='SQS Queue to send netkan metadata for inflation',
+    required=True,
+)
+@click.option(
+    '--timeout', default=300, envvar='SQS_TIMEOUT',
+    help='Reduce message visibility timeout for testing',
+)
+@click.option(
+    '--ckan-meta', envvar='CKANMETA_REMOTE',
+    help='Path/URL/SSH to CKAN-meta repo for mirroring',
+    required=True,
+)
+@click.option(
+    '--ia-access', envvar='IA_access',
+    help='Credentials for Internet Archive',
+    required=True,
+)
+@click.option(
+    '--ia-secret', envvar='IA_secret',
+    help='Credentials for Internet Archive',
+    required=True,
+)
+@click.option(
+    '--ia-collection', envvar='IA_collection',
+    help='Collection to put mirrored mods in on Internet Archive',
+    required=True,
+)
+def mirrorer(queue, timeout, ckan_meta, ia_access, ia_secret, ia_collection):
+    Mirrorer(
+        init_repo(ckan_meta, '/tmp/CKAN-meta'),
+        ia_access, ia_secret, ia_collection
+    ).process_queue(queue, timeout)
+
+
+@click.command()
+@click.option(
+    '--ckan-meta', envvar='CKANMETA_REMOTE',
+    help='Path/URL/SSH to CKAN-meta repo for mirroring',
+    required=True,
+)
+@click.option(
+    '--ia-access', envvar='IA_access',
+    help='Credentials for Internet Archive',
+    required=True,
+)
+@click.option(
+    '--ia-secret', envvar='IA_secret',
+    help='Credentials for Internet Archive',
+    required=True,
+)
+@click.option(
+    '--ia-collection', envvar='IA_collection',
+    help='Collection to put mirrored mods in on Internet Archive',
+    required=True,
+)
+def mirror_purge_epochs(ckan_meta, ia_access, ia_secret, ia_collection):
+    Mirrorer(
+        init_repo(ckan_meta, '/tmp/CKAN-meta'),
+        ia_access, ia_secret, ia_collection
+    ).purge_epochs()
 
```

Comment:
I think we need a dry run option for this. It'd be nice to know what actions it's performing before committing to the action (being something we'll probably only run once).

Reply:
Good idea, added this.